Manager/FormManager.php

```diff
@@ -100,18 +102,93 @@ public function save($data, $locale = null, $id = null)
         $translation->setDeactivateCustomerMails(self::getValue($data, 'deactivateCustomerMails'));
         $translation->setChanged(new \DateTime());
 
-        // Add Translation to Form
+        // Add Translation to Form.
         if (!$translation->getId()) {
             $translation->setForm($form);
             $form->addTranslation($translation);
         }
 
-        // Set Default Locale
+        // Set Default Locale.
         if (!$form->getId()) {
             $form->setDefaultLocale($locale);
         }
 
-        // Fields
+        // Fields.
+        $this->updateFields($data, $form, $locale);
+
+        // Receivers.
+        $this->updateReceivers($data, $translation);
+
+        // Save.
```

Comment:
the dot makes only sense if the comment will be a sentence. this should also be the intention of this definition `Everytime a dot at the end`. We have the definition to add it in the phpdoc but then it has to make a sense. but in the normal comment you can decide to use only bullet-points without dot or sentences with a dot.